Data/Map/Lazy.hs

```diff
@@ -184,9 +189,6 @@ module Data.Map.Lazy (
 
     -- ** Lists
     , toList
-    , fromList
-    , fromListWith
-    , fromListWithKey
```

Comment:
Hrmmmm.. Maybe.

Reply:
I thought these fit more with construction because they allow you to build a map from some other value. I think of conversions as converting a map to some other value, the opposite of construction.